client/state/ui/guided-tours/actions.js

```diff
@@ -21,14 +21,16 @@ import { getPreference } from 'state/preferences/selectors';
  * @param {Object} options Options object, see fn signature.
  * @return {Object} Action object
  */
-export function showGuidedTour( { shouldShow, shouldDelay = false, tour = 'main' } ) {
+export function showGuidedTour( { shouldShow, shouldDelay = false, tour } ) {
 	const showAction = {
 		type: GUIDED_TOUR_SHOW,
 		shouldShow,
 		shouldDelay,
 		tour,
 	};
 
+	// TODO(mcsf): track this *somewhere*, now that we'll stop relying on this
+	// action for launching tours
```

Comment:
Any idea how we'd do this? Maybe a `isFirstShow` in the GT selector, then `<GuidedTours>` can fire off the action.